client/src/theme.js

```diff
@@ -36,6 +42,43 @@ let theme = createTheme({
         },
       ],
     },
+    MuiTextField: {
+      defaultProps: {
+        inputProps: {
+          style: { width: '100%', border: 'none', color: 'black' },
+        },
+        fullWidth: true,
+      },
+    },
+    MuiInputBase: {
+      styleOverrides: {
+        root: {
+          '& .MuiInputBase-input': {
+            color: 'black',
+            fontFamily: 'Source Sans Pro',
+            fontSize: '18px',
+            fontWeight: '500',
+            border: 'none',
+            '&::placeholder': {
+              color: '#757575',
```

Comment:
Suggest we move to only using color variables istead of hardcoded values, though it's outside the scope of this PR